modules/ipi-install-configuring-storage-on-nodes.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-storage-on-nodes_{context}"]
+= Optional: Configuring storage on nodes
+
+You can make changes to operating systems on {product-title} nodes by creating `MachineConfig` objects that are managed by the Machine Config Operator (MCO).
+
+The `MachineConfig` specification includes an ignition config for configuring the machines at first boot. This config object can be used to modify files, systemd services, and other operating system features running on on {product-title} machines.
+
+.Procedure
+
+Use the ignition config to configure storage on nodes. The following `MachineSet` manifest example demonstrates how to add a partition to a device on a primary node. In this example, apply the manifest before installation to have a partition named `recovery` with a size of 16 GiB on the primary node.
+
+. Create the new `MachineConfig` manifest as shown in the following example:
++
+[source,terminal]
```

Comment:
I wonder if it would be best to turn this into an actual command (otherwise, I think it would be [source,yaml]. Sometimes we do something like the following: 

```
$ cat <<EOF| oc create -f -
---
---
EOF
```

Sometimes, though, we just say, like: 
```

Create the `MachineConfig` object:
$ oc create -f <file_name>.yaml
```

or 

`. To create a `MachineConfig` object inside of a config map in the management cluster, enter the following information:
+
[source,yaml]
---
`
I'm not sure what the best situation here would be to do, given the context. Your dev might know for sure!

Reply:
@danielerez @vkolodny, what do you suggest for this section?

Reply:
@ktothill probably not a good idea to use `oc create` just yet since this is pre-installation so a cluster won't be available. I'd go with creating the file, similar to step 1 in the [Optional: Deploying routers on worker nodes](https://docs.openshift.com/container-platform/4.12/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#deploying-routers-on-worker-nodes_ipi-install-installation-workflow). So let's give the file an actual name, such as `custom-partitions.yaml`. For example:

`1. Create a custom-partitions.yaml file and include a MachineConfig object that contains your partition layout:`

Reply:
Updated As Suggested.